### PR TITLE
Assignemt1 and Assignment 2

### PR DESCRIPTION
**Pull request by AmrutaDhage20**
Assignment 1: Check if All Characters Have Equal Number of Occurrences
Assignment 2: Find Destination City

### CODE REVIEW

File: src/com/encora/javaAssignments/PrimitiveDataTypes.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		boolean g = true;
+		
+		System.out.println("char: " +a);
+		System.out.println("byte: " +b);
+		System.out.println("short: " +c);
+		System.out.println("int: " +d);
+		System.out.println("flaot: " +e);
+		System.out.println("double: " +f);
+		System.out.println("boolean: " +g);
+
```
Comment (rohitlamsonge66) [inline src/com/encora/javaAssignments/PrimitiveDataTypes.java]: Remove the extra space.

File: src/com/encora/javaAssignments/PrimitiveDataTypes.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package com.encora.javaAssignments;
+
+public class PrimitiveDataTypes {
+
+	public static void main(String[] args) {
+	
+		// declaration of primitive data types
+		char a = 'A';
+		byte b = 2;
```
Comment (rohitlamsonge66) [inline src/com/encora/javaAssignments/PrimitiveDataTypes.java]: Give proper understandable name to variable.

File: src/com/encora/javaAssignments/NonPrimitiveDataTypes.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+		
+		String str1 = new String ("Test The String"); // declaration of string
+		System.out.println("Second String: " +str1);
+		
+		int arr[]; //declaration of array
+		arr = new int[2]; //array integer is till index is 2
+		arr[0]= 0;
+		arr[1]= 1;
+		
+		//System.out.println(arr); // prints the whole memory location value
```
Comment (rohitlamsonge66) [inline src/com/encora/javaAssignments/NonPrimitiveDataTypes.java]: don't push not required commented code.

File: src/com/encora/javaAssignments/ImplicitDataTypeConversion.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+		int a = 100;
+		System.out.println("Int representation: "+a);
+		
+		long b=a;
+		System.out.println("Long representation: "+b);
+		
+		float c = b;
+		System.out.println("Float representation: "+c);
+
+
```
Comment (rohitlamsonge66) [inline src/com/encora/javaAssignments/ImplicitDataTypeConversion.java]: Write the question before writing the program in commented format.

File: src/com/encora/javaAssignments/ImplicitDataTypeConversion.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package com.encora.javaAssignments;
+
+public class ImplicitDataTypeConversion 
+{
+
+	public static void main(String[] args) 
+	{
+		int a = 100;
```
Comment (rohitlamsonge66) [inline src/com/encora/javaAssignments/ImplicitDataTypeConversion.java]: Use method to write the repeatable code.

File: src/com/encora/javaAssignments/PrimitiveDataTypes.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+		System.out.println("byte: " +b);
+		System.out.println("short: " +c);
+		System.out.println("int: " +d);
+		System.out.println("flaot: " +e);
+		System.out.println("double: " +f);
+		System.out.println("boolean: " +g);
+
+
+	}
+
```
Comment (rohitlamsonge66) [inline src/com/encora/javaAssignments/PrimitiveDataTypes.java]: Use the comments for all the programs.

File: src/com/encora/javaAssignments/GoodString.java [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+          } else {
+              charCount.put(s.charAt(i), 1);
+          }
+          System.out.println("\nChar - " + s.charAt(i));
+      }
+       System.out.println("\n charCount - " + charCount);
+
+       //Stage 2 - check if all counts doesn't match then its a not a good string
+      int res = 0;
+      boolean isGoodString = true;
```
Comment (amitsahu8) [inline src/com/encora/javaAssignments/GoodString.java]: default boolean should be false and check of true condition

File: src/com/encora/javaAssignments/GoodString.java [hunk 8]
```diff
@@ -0,0 +1,38 @@
+package com.encora.javaAssignments;
+
+import java.util.HashMap;     
+
+public class GoodString {
+  public static void main(String args[])
+  {
+      String s = "abacbdc";
```
Comment (amitsahu8) [inline src/com/encora/javaAssignments/GoodString.java]: use meaningful variable name

File: src/com/encora/javaAssignments/DestinationCity.java [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+//Assignment2 -Find Destination City
+
+import java.util.*;
+
+public class DestinationCity {
+	public static String destCity(String[][] input) {
+		int i;
+
+		HashMap<String, Integer> outdegree = new HashMap<String, Integer>();
+		if (input.length == 1) {
```
Comment (amitsahu8) [inline src/com/encora/javaAssignments/DestinationCity.java:29]: try without any datatype in hashmap

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

**Review by rohitlamsonge66 — APPROVED** [review]

**Pull request merged**